tests/unit/test_utils.py

```diff
@@ -562,31 +562,40 @@ def test_run_big():
     assert len(stdout) == 200000
 
 
-def test_get_distgit_handler():
-    for wrong_remotes in [[], ["blah"]]:
-        with pytest.raises(tmt.utils.GeneralError):
-            tmt.utils.get_distgit_handler([])
-    # Fedora detection
-    returned_object = tmt.utils.get_distgit_handler("""
-        remote.origin.url ssh://lzachar@pkgs.fedoraproject.org/rpms/tmt
-        remote.lzachar.url ssh://lzachar@pkgs.fedoraproject.org/forks/lzachar/rpms/tmt.git
-        """.split('\n'))
-    assert isinstance(returned_object, tmt.utils.FedoraDistGit)
-    # CentOS detection
-    returned_object = tmt.utils.get_distgit_handler("""
-        remote.origin.url git+ssh://git@gitlab.com/redhat/centos-stream/rpms/ruby.git
-        """.split('\n'))
-    assert isinstance(returned_object, tmt.utils.CentOSDistGit)
-
-
-def test_FedoraDistGit(tmpdir):
-    # Fake values, production hash is too long
-    tmpdir.join('sources').write(
-        'SHA512 (fn-1.tar.gz) = 09af\n')
-    tmpdir.join('tmt.spec').write('')
-    fedora_sources_obj = tmt.utils.FedoraDistGit()
-    assert [("https://src.fedoraproject.org/repo/pkgs/rpms/tmt/fn-1.tar.gz/sha512/09af/fn-1.tar.gz",  # noqa: E501
-            "fn-1.tar.gz")] == fedora_sources_obj.url_and_name(cwd=tmpdir)
+class Test_dist_git_source:
```

Comment:
Why are you adding the class here? I remember @happz just recently started removing classes from unit tests as they are not needed.

Reply:
Easier to navigate and filter test cases. But frankly I don't care so I'll undo